tagtester/main.py

```diff
@@ -61,10 +58,11 @@ def capture_tags():
                     pb.config = config
                     taghtml = tags[cid][s][t]
                     pb.create()
-                    browser_errors[pb.tagname] = capture.capture_tag(
-                        driver=driver, tag_html=taghtml,
+                    browser_errors[pb.tagname] = tagcapture.capture_tag(
+                        tag_html=taghtml,
                         output_path=pb.tagimage, tagtype=t)
-        driver.quit()
+                    LOGGER.info('Captured {}'.format(pb.tagimage))
+        tagcapture.stop()
```

Comment:
You probably want this in a `finally:` block so it happens if there's an exception. Or rename it to `close()` and use `contextlib.closing`, like:

```python
from contextlib import closing
with closing(capture.TagCapture.from_config(config)) as tagcapture:
    # do your stuff with tagcapture, confident that it will be closed
    # when you are done, regardless of what happens.
```